sources/packages/backend/apps/api/src/route-controllers/application-restriction-bypass/_tests_/e2e/application-restriction-bypass.aest.controller.getApplicationRestrictionBypasses.e2e-spec.ts

```diff
@@ -0,0 +1,136 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import {
+  E2EDataSources,
+  RestrictionCode,
+  createE2EDataSources,
+  createFakeUser,
+  saveFakeApplication,
+  saveFakeApplicationRestrictionBypass,
+} from "@sims/test-utils";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import * as request from "supertest";
+import {
+  ApplicationStatus,
+  OfferingIntensity,
+  RestrictionActionType,
+  User,
+} from "@sims/sims-db";
+
+describe("ApplicationRestrictionBypassAESTController(e2e)-getApplicationRestrictionBypasses.", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let sharedMinistryUser: User;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    sharedMinistryUser = await db.user.save(createFakeUser());
+  });
+
+  it("Should get a list of application restriction bypasses for a submitted part-time application when there is no restriction bypass associated with it.", async () => {
```

Comment:
The title says `there is no restriction bypass associated with it`. Please adjust the title.

Reply:
Updated. There are three test cases. One of them is to obtain an empty array for a submitted application. Another one is to obtain an empty array for a draft application. The last one is to obtain an array of bypasses for a submitted application.